packages/manager/modules/octavia-load-balancer/src/octavia-load-balancer.routing.js

```diff
@@ -1,10 +1,17 @@
 export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('octavia-load-balancer', {
-    url: '/octavia-load-balancer',
+    url: '/pci/projects/{projectId:[0-9a-zA-Z]{32}}/octavia-load-balancer',
     template: '<div data-ui-view></div>',
     redirectTo: 'octavia-load-balancer.onboarding',
     resolve: {
-      breadcrumb: ($translate) => $translate.instant('octavia_lb'),
+      projectId: /* @ngInject */ ($transition$) =>
+        $transition$.params().projectId,
+      project: /* @ngInject */ (OvhApiCloudProject, projectId) =>
+        OvhApiCloudProject.v6().get({
+          serviceName: projectId,
+        }).$promise,
```

Comment:
I think it's better to use the v6 api (GET /cloud/project/{serviceName}) directly with $http (in a service) than to use an old ovh-api-service library (I think it's deprecated, by the way).